### ISSUE TITLE
Pending changes window has crashed :(

### ISSUE THREAD

**Issue by PureKrome** [jzoss/Git-Source-Control-Provider#15]
Not sure what I did to repo this.. but .. 

I just clicked the `Pending Changes` window and was show this error page/message..

```
An exception was encountered while constructing the content of this frame.  This information is also logged in "C:\Users\Pure Krome\AppData\Roaming\Microsoft\VisualStudio\14.0\ActivityLog.xml".

Exception details:
System.ArgumentNullException: Value cannot be null.
Parameter name: key
   at System.Collections.Concurrent.ConcurrentDictionary`2.TryGetValue(TKey key, TValue& value)
   at GitScc.RepositoryManager.GetTrackerForPath(String filename, Boolean createTracker)
   at GitScc.PendingChangesToolWindow.OnToolWindowCreated() in C:\DevelExamples\Git-Source-Control-Provider\GitSccProvider\PendingChangesToolWindow.cs:line 66
   at Microsoft.VisualStudio.Shell.Package.CreateToolWindow(Type toolWindowType, Int32 id, UInt32 flags)
   at Microsoft.VisualStudio.Shell.Package.CreateToolWindow(Type toolWindowType, Int32 id, ProvideToolWindowAttribute tool)
   at Microsoft.VisualStudio.Shell.Package.FindToolWindow(Type toolWindowType, Int32 id, Boolean create, ProvideToolWindowAttribute tool)
   at Microsoft.VisualStudio.Shell.Package.CreateToolWindow(Guid& toolWindowType, Int32 id)
   at Microsoft.VisualStudio.Shell.Package.Microsoft.VisualStudio.Shell.Interop.IVsToolWindowFactory.CreateToolWindow(Guid& toolWindowType, UInt32 id)
   at Microsoft.VisualStudio.Platform.WindowManagement.WindowFrame.ConstructContent()
```

le sniffles.

**Comment by jzoss** [comment]
So.. I tried to fix this.. I think I found it, but I was not able to recreate it.. Anyway can you help me out.. check out this link [build]( https://ci.appveyor.com/api/buildjobs/3362l96s4v4aexbe/artifacts/GitSccProvider/bin/Release/GitSccProvider.vsix) I put the fix in there.. So.. there is also a few other changes in there, if you want to tell me what you think.. The theme switching is fixed.. I added a new feature to update you .gitignore ... but lots of it is making it much faster.. The autoupdate loop is gone.. Now it's all events.. Anyway, check it out.. if it's terrible I will branch the release and put the fix in there.

**Comment by PureKrome** [comment]
Just installed it .. now trying it out. :clap:

**Comment by jzoss** [comment]
Fixed in 1.5 release
